### ISSUE TITLE
Change misleading repository name

### ISSUE THREAD

**Issue by belvederef** [inrupt/solid-react-sdk#32]
I feel like the name of this repository is confusing and misleading. I believe it should be part of a more generic `/docs` repository since no code is hosted here, and that issues are opened on the respective projects since it gets hard to follow and to keep up to date.

**Comment by ThisIsMissEm** [comment]
Hi @belvederef, I'm going to archive this repo as it's been superseded by https://github.com/inrupt/solid-ui-react

Hopefully that'll clear this up. Thanks for mentioning this
